docs/IR.md

```diff
@@ -85,42 +86,30 @@ lowered into the mid-level IR in a phase that's called "IRGen" (stands for IR
 generation). This is a one-to-many translation where each operator is translated
 into one or more instructions.
 
-### Variable Visibility
```

Comment:
We still have WeightVar Visibility -- should we keep a section about this? Actually, maybe we can remove Visibility from WeightVar too (and so remove it entirely)? I don't think there's any difference at this point between Mutability vs. Visibility  -- I believe Constant is always Private, and Mutable is always Public.

Reply:
Good idea. I submitted #1811 to remove the visibility.